recipe_scrapers/__init__.py

```diff
@@ -138,7 +138,7 @@ class WebsiteNotImplementedError(NotImplementedError):
     pass
 
 
-def scrape_me(url_path):
+def scrape_me(url_path, **kwargs):
```

Comment:
This seems a little ugly / risky, but does potentially allow for other scraper options in future too.  I don't love usage of `*args` (so it's omitted here) since ordering of arguments can be troublesome especially for backwards-compatibility.

This seems like the kind of API decision that *could* be difficult to roll back in future.

Reply:
To be on the safe side, I've decided to introduce an experimental `harvest` method that will exist alongside the `scrape_me` function.  That means we can leave `scrape_me` unmodified in this pull request, and callers won't see any effect (nor be able to access the additional options using it)